oc_ldap_user_locker/locker.py

```diff
@@ -160,18 +161,46 @@ def _check_user_conf(self, user_rec, conf):
         _matched_attributes = 0
         for _attrib in conf['condition_attributes'].keys():
             logging.debug("Comparing attribute: '%s'" % _attrib)
-            _vals_from_rec = user_rec.get_attribute(_attrib)
             _match_conf = conf['condition_attributes'][_attrib]
+            
+            if "." not in _attrib:
+                _vals_from_rec = user_rec.get_attribute(_attrib)
+
+                if not self._compare_attribute(_vals_from_rec, _match_conf):
+                    logging.debug("Failed on attribute: '%s'" % _attrib)
+                    return None
+            else:
+                # attribute containing references to objects + attribute to search for the values in these objects
+                _attrib_pair = _attrib.split(".")
```

Comment:
Here we shall fail if someone puts `a.b.c`
Suggestion is to limit number of splits: `_attrib_main, _attrib_sub = _attrib.split(".", 1)`

And then use `_attrib_main` and `_attrib_split` as separate more readable values than `attrib_pair[0], attrib_pair[1]`